charts/helsenorge-application/README.md

```diff
@@ -1,10 +1,12 @@
 
 
+
+
 # helsenorge-applikasjon
 
 Helm chart for installere en helsenorge-applikasjon på kubernetes. En helsenorge-applikasjon dekker typene API, WebApp, Service, Batch. Dvs, applikasjoner som utfører arbeid kontinuerlig.
 
-![Version: 0.0.43](https://img.shields.io/badge/Version-0.0.43-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square)
+![Version: 0.0.44](https://img.shields.io/badge/Version-0.0.44-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) 
```

Comment:
Gjør dette noe funksjonelt, utenom å tydeliggjøre versjon? Kan vurderes fjernes for å slippe å vedlikeholde, bare en tanke.

Reply:
Den gir ikke så mye verdi nei. Kan fjerne den.

Reply:
done